ably.xcworkspace/contents.xcworkspacedata

```diff
@@ -2,6 +2,9 @@
 <Workspace
    version = "1.0">
    <FileRef
+      location = "group:ablySpec/ReadmeExamples.swift">
```

Comment:
BTW, use this https://github.com/github/gitignore/blob/master/Objective-C.gitignore